packages/amplify-category-api/src/provider-utils/awscloudformation/base-api-stack.ts

```diff
@@ -75,7 +77,7 @@ export abstract class ContainersStack extends cdk.Stack {
   protected readonly awaiterS3Key: string;
 
   constructor(scope: Construct, id: string, private readonly props: ContainersStackProps) {
-    super(scope, id);
+    super(scope, id, { synthesizer: new cdk.LegacyStackSynthesizer() });
```

Comment:
Without `LegacyStackSynthesizer` the template depends on CDK bootstrap. Which is missing as we don't create cdk bootstraps in the account. This removes that dependency.